### DISCUSSION TITLE
Swagger API spec option ? [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2084] [Discord Threads]
> 🙋‍♂️ **M. Kalatchev**  
##   
![image0](https://cdn.discordapp.com/attachments/661905455894888493/971552144350146650/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Sorry! I resolved it by myself. I did two things - upgrade to latest version and set NC_PUBLIC_URL. Obviously this field is mandatory to have this link working 🙂   
####    
#### (answered by <@618257341703520267>)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/971552145092542504)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
